viz/charts/linkerd-viz/templates/tap.yaml

```diff
@@ -31,22 +32,24 @@ apiVersion: apps/v1
 metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
+    linkerd.io/inject: enabled
+    config.linkerd.io/proxy-await: "enabled"
```

Comment:
If tap will get injected through the injector, do we need to be explicit about `proxy-await`? It should be the default injector behaviour to add await now post 2.10.1 so we might be able to remove it?